src/components/Loader.vue

```diff
@@ -395,4 +395,23 @@ export default Loader;
         max-width: 70%;
         opacity: 1;
     }
+
+    .loader.exit-transition {
+        /* animation: exit-transition 600ms 1s; */
+    }
+
+    @keyframes exit-transition {
+        from { transform: scale(1); opacity: 1; }
+        80%  { opacity: 0; }
+        to   { transform: scale(0); opacity: 0; }
+    }
```

Comment:
...until here, I guess?

Reply:
The whole `exit-transition` class + logic can then also be removed, as currently it only affects `.success .icon-row` which is only visible on success anyways.